teacher.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'person'
```

Comment:
Watch out for your indentation! Make sure that it is consistent throughout your file and throughout your entire project so that it is clean and easy to read.